kinto/core/testing.py

```diff
@@ -34,9 +34,9 @@ def __init__(self, *args, **kwargs):
         self.authn_type = 'basicauth'
         self.prefixed_userid = 'basicauth:bob'
         self.effective_principals = [
-            self.prefixed_userid,
             'system.Everyone',
-            'system.Authenticated']
+            'system.Authenticated',
+            'bob']
```

Comment:
This `effective_principals` value truely respects what Pyramid would return. The previous list was wrong, there was no way we could obtain a prefixed_userid in the principals with the previous code.